.github/workflows/build.yml

```diff
@@ -10,6 +10,17 @@ on:
     branches: [ master ]
 
 jobs:
+  verify-python:
+    name: Python code verification (src)
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install dependencies
+        run: |
+          pip install black
+      - name: Verify code (python w/black)
+        run: |
+          black --check -t py36 src/
```

Comment:
what is the `-t py36` for?

Reply:
This stands for the Python version. PTF should drop support for Python2, but before it happens, Black will automatically resolve the Python version. 

When PTF will be fully Python3 compatible, there should be set a min target for interpreters in `makefile` command.